components/faq.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Document</title>
+    </head>
+    <body>
+        <main>
+        <p>Need Help?</p>
+        <input type="search" name="searchFaq">
+       
+        <h1>FAQS</h1>
+        <section class="question">
+            Question 1
```

Comment:
Same as the previous comment, plain text shouldn't be without a html tag. That includes ALL section Question X texts and the footer one. Also for semantic reasons it's a great idea to use <hX> instead of just <p>, as all of them seems to be titles.